api/app/Integrations/Handlers/AbstractIntegrationHandler.php

```diff
@@ -26,6 +27,25 @@ public function __construct(
         $this->form = $event->form;
         $this->submissionData = $event->data;
         $this->integrationData = $formIntegration->data;
+        $this->mailer = config('mail.default');
```

Comment:
These changes but I think this should not be on the abstract integration class itself as many integrations are not mails, instead we should probably create a new abstract class extending this one for mail integrations, and add this there

Reply:
And then our email integrations would extend this new class